doc/03_research/03_planning/basics.md

```diff
@@ -1,49 +1,57 @@
 # Grundrecherche im Planing
 
 ## Authors
+
 Simon Erlbacher, Niklas Vogel
 
 ## Datum
+
 15.11.2022
 
```

Comment:
a TOC would have been nice for better navigation, following the wiki_template.md